Header HEADER
string NAME

uint8 SET_COMMAND # 1 : START, 2 : STOP, 3 : WAIT FINISH MESSAGE, 4 : READ SAVED DATA

char SET_LINE
int32 SET_DISCHARGE
int32 SET_CAP_TIME
